runner/src/python.rs

```diff
@@ -337,6 +337,25 @@ impl PyEnv {
         let module = PyModule::import(py, PyString::new(py, &path.module().to_string()))?;
         module.getattr(PyString::new(py, path.name()))
     }
+
+    pub fn find_spec_search_locations(&self, py: Python, path: &PathStr) -> PyResult<Vec<PathBuf>> {
+        let importlib = py.import(intern!(py, "importlib"))?;
+        let spec = importlib
+            .getattr(intern!(py, "util"))?
```

Comment:
It does not work for me:

```
Whoops! AttributeError: module 'importlib' has no attribute 'util' (This isn't your fault)
```

I have installed `Python 3.12.3 (main, Apr  9 2024, 08:09:14) [Clang 15.0.0 (clang-1500.3.9.4)]`

Reply:
haha python https://discuss.python.org/t/python3-11-importlib-no-longer-exposes-util/25641 . try it now (: